### DISCUSSION TITLE
ETKDG conformer output with same seed changes from version 2024.3.4 to 2024.3.5, is this expected?

### DISCUSSION THREAD

**Discussion by andrewtarzia** [rdkit/rdkit#7841] [Q&A] [answered]
We use ETKDGv2 with stk, and a constant random seed that has provided the same conformers for a while, but has changed since 2024.3.5.

Looking through the release notes, this may be expected with changes in the distance geometry code?

**Comment by greglandrum** [comment, marked as answer]
There was a bug introduced in the ETKDG code in the 2024.03.5 release. This has been fixed in the 2024.03.6 release (https://github.com/rdkit/rdkit/pull/7729), but unfortunately there have been problems making the conda-forge builds for that release work. I think that I have that cleared up, so keep any eye out for a 2024.03.6 version in conda-forge in the next day or two.

**Reply by andrewtarzia** [reply]
Ok, thank you for the heads up!

**Reply by andrewtarzia** [reply]
Apologies if I previously missed that announcement.

**Reply by greglandrum** [reply]
> Apologies if I previously missed that announcement.

There was no formal announcement, just the bug tag on the PR fixing the problem and the corresponding line in the release notes.

**Reply by greglandrum** [reply]
@andrewtarzia the new release is up now

**Reply by andrewtarzia** [reply]
Thank you!
